README.md

```diff
@@ -1,5 +1,5 @@
 <h1 align="center">odysseyn1x</h1>
-<h3 align="center">Linux-based distro that lets you install checkra1n, odysseyra1n and Project Sandcastle.</h3>
+<h3 align="center">Linux-based distro that lets you install Checkra1n, Odysseyra1n and Project Sandcastle.</h3>
```

Comment:
I think checkra1n and odysseyra1n don't start with a capital letter. Check these links out: https://www.reddit.com/r/jailbreak/wiki/checkra1n#wiki_switch_from_checkra1n_to_odysseyra1n